tests/test_self.py

```diff
@@ -584,6 +586,18 @@ def test_stdin_syntaxerror(self) -> None:
             )
             assert mock_stdin.call_count == 1
 
+    @pytest.mark.skipif(
+        PY310_PLUS, reason="Not a syntax error for python 3.10 or superior"
+    )
+    def test_astroid_syntax_error(self, tmp_path) -> None:
```

Comment:
```suggestion
    def test_astroid_syntax_error(self, tmp_path: TempPathFixture?) -> None:
```

Reply:
Hurgh, no that's my bad. It's a pathlib.Path.